integration/utils/scripts.go

```diff
@@ -127,17 +127,17 @@ access(all) contract MyFavContract {
     }
 
     // items
-    access(all) event NewItemAddedEvent(id: UInt32, metadata: {String: String})
+    access(all) event NewItemAddedEvent(id: UInt32, metadata: {Name: Name})
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig